lib/tests/streamlit/runtime/caching/hashing_test.py

```diff
@@ -74,6 +75,15 @@ def test_int(self):
         self.assertNotEqual(get_hash(2**7), get_hash(2**7 - 1))
         self.assertNotEqual(get_hash(2**7), get_hash(2**7 + 1))
 
+    def test_uuid(self):
+        uuid1 = uuid.uuid4()
+        uuid1_copy = uuid.UUID(uuid1.hex)
+        uuid2 = uuid.uuid4()
+
+        self.assertEqual(get_hash(uuid1), get_hash(uuid1_copy))
+        self.assertNotEqual(id(uuid1), id(uuid1_copy))
+        self.assertNotEqual(get_hash(uuid1), get_hash(uuid2))
```

Comment:
Are we just supporting uuid4()? (I don't know if this is necessary, but I'm wondering if it makes sense to test for the various other UUID formats. Do they all use the same `uuid.UUID` type, in which case it seems much less important?)